public/assets/js/profile-controller.js

```diff
@@ -0,0 +1,35 @@
+$("#profileUpdate").on("click", function() {
+  $("#profileEventForm").hide()
```

Comment:
if this was annoying to write out .hide() over and over again, an alternative could be comma separating these.

$('#profileEventForm, #profileClassForm, #profileResForm, #profileContactForm').hide();

does the same outcome, it just selects all those elements at once, and hides, vs technically selecting one, then hide, then the next, then hide, and so on.